sharpd/sharp_vty.c

```diff
@@ -176,6 +191,13 @@ DEFPY (install_routes,
 	}
 	sg.r.orig_prefix = prefix;
 
+	vrf = vrf_lookup_by_name(name ? name : VRF_DEFAULT_NAME);
```

Comment:
One small nit... to simplify the code you could do this:
```c
    if (!name)
        name = VRF_DEFAULT_NAME;
```
This way `name` would always contain the VRF name, and you wouldn't need to repeat `name ? name : VRF_DEFAULT_NAME` whenever necessary. IIRC we do this in staticd.